eclair-core/src/main/scala/fr/acinq/eclair/transactions/Transactions.scala

```diff
@@ -100,7 +100,7 @@ object Transactions {
     case object Remote extends TxOwner
   }
 
-  sealed trait TransactionWithInputInfo {
+  trait TransactionWithInputInfo {
```

Comment:
This change is unrelated to the `UnkownMessage`, I think it should be removed from this PR.

I don't think it's useful at all anyway, `checkSpendable` is a one-line function that calls `bitcoin-lib`, you should just replicate it in the `PeerSwap` plugin instead of depending on `TransactionWithInputInfo`.

Reply:
Fixed in [8606def](https://github.com/ACINQ/eclair/commit/8606defdd703896982cd024185fa4ffa9bf6b021).